lib/definitions/livesync.d.ts

```diff
@@ -0,0 +1,220 @@
+interface IFSWatcher extends NodeJS.EventEmitter {
```

Comment:
Can you add a comment at the top that these interfaces are copied from Node.js and Chokidar's types